### ISSUE TITLE
1.7.8.5 logs Error (3) Service Unavailable 503 when accessing Dashboard while shop is in maintenance mode

### ISSUE THREAD

**Issue by Crwmy** [PrestaShop/PrestaShop#28449]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

I recently installed PrestaShop 1.7.8.5 via Click and Build from 1&1 IONOS to test it as possible replacement option.

When testing I noticed that PrestaShop logged very often an Error (3) Service Unavailable (503). Through testing I narrowed it down. I can reproduce it when accessing the Dashboard. As I did not have had any visible issues (white pages) I thought it could be a php timeout. But nothing was logged when PHP error logging was activated via php.ini and debug mode also threw no errors. 

But when I deactivated maintenance mode the error logging stopped (and continued when I activated it again).

For a newbie like me this is highly confusing when trying to evaluate if the system runs smooth or not, because it is not clear that the error comes from maintenance mode when the shop is not reachable. 

### Expected behavior

No Error (3) Service Unavailable (503) logging while shop is in maintenance mode

### Steps to reproduce

1. Go to BO Shop Parameters -> General -> Maintenance -> Disable Shop
2. Go to BO Dashboard

(3. Go to BO Advanced Parameters -> Logs)

### PrestaShop version(s) where the bug happened

1.7.8.5

### PHP version(s) where the bug happened

7.4.29

**Comment by hibatallahAouadni** [comment]
Hello @Crwmy 

I didn't manage to reproduce the issue with the provided infos, see the attached screenshot below:

![image](https://user-images.githubusercontent.com/23308427/167467971-a080d134-90f9-4c74-9654-87772b4d4d94.png)

Could you please provide more details and a screen record :pray: 
Waiting for your feedback.

Thanks!

**Comment by Crwmy** [comment]
Hi @hibatallahAouadni

Thanks for looking into this curious issue! 

I have attached a screen recording that I edited with annotations in hope to give enough additional context. Slightly nervous that I might overexplain.  

https://user-images.githubusercontent.com/91319158/167499556-5eb5d974-88f0-4af3-9008-7d4f577a0672.mp4

**Comment by marwachelly** [comment]
Hello @Crwmy,

I didn't manage to reproduce your issue with the provided information.

https://user-images.githubusercontent.com/89604350/167593709-db5cf565-aa3d-4aa6-b119-8067027e43ab.mp4

Did I miss some thing? 

Waiting for your feedback!

Thanks,

**Comment by Crwmy** [comment]
Hi @marwachelly,

Thank you for taking the time and looking at this issue and providing a screen recording.
It does not seem that you miss a thing from what the screen recording shows. 

I was confused and thought the error lies with me and actually screwed something up. So I conducted further testing. And found the culprit: PrestaShop Metrics!

I have installed a separate Click & Build Version, this time without Demo Store content (just to be sure), and tested again immediately after login and the error did not occur. At this point I was certain that if I have done something while configure the shop I'll come along unintentionally again.

So I scrapped my Evaluation Install that I have had populated with some of the actual products we want to sell and re-installed it - also without Demo Store content. No error after initial testing. So far so good.

Interestingly, I got a new error: "Data from PrestaShop Addons is invalid, and cannot fallback on cache". Which I could pinpoint to the Module Manager. I was surprised to find out that even though this was a fresh install in a fresh database, the Module Manager thought I was logged in to PrestaShop Addons. This caused the error. After "logging out" and re-logging in the "Data from..." errors stopped. 

Then 8 Updates were shown. One of them was PrestaShop Metrics **3.0.0**. I installed all and then the 503 error started again when visiting the Dashboard. PrestaShop Metrics also said it needed an update, but there was none visible in the Update Pane of the Module Manager. I thought I try deactivating PrestaShop Metrics and then saw that the Module was on version **3.1.0**. So it was apparent that there was an inconsistency. I deactivated Metrics and  the 503 errors stopped.  

I checked on PrestaShop Addons and saw that the latest version was **3.1.0**. I downloaded the zip and uploaded it via Module Manager. After successfully installing it, I activated it and the 503 errors started again (and stopped when deactivating PrestaShop Metrics).  

I'm not sure what causes this, but it is independent from the Browser (tested with Microsoft Edge 101 and Firefox 100) with deactivated NoScript.

Maybe this has also something to do with IONOS and their Click & Build install routine, because in Advanced Parameters -> Information "config/defines.inc.php" is the only listed under Updated files and there seem to be additions as when I tried debugging and activated the Profiler - which was said to be in line 43, it was at 50 or so in my file. And install.txt is missing, but I think that has no effect.

From observation it seems to be connected to maintenance mode and that there is something PrestaShop Metrics doesn't like when the Shop is behind that maintenance notification page and can't be reached, But I'm guessing here.
